packages/website/blog/2023-07-09-announcing-typescript-eslint-v6.md

```diff
@@ -628,19 +628,23 @@ import * as TSESLint from '@typescript-eslint/utils/dist/ts-eslint';
 
 That presents a few issues for developers:
 
-- It can be unclear which of many potential import paths to use
-- TypeScript sometimes suggests importing types or values meant to be private
-- Consumers using deep import paths can be broken by internal refactors that rename files
+- It can be unclear which of many potential import paths to use.
+- TypeScript sometimes suggests importing types or values meant to be private.
+- Consumers using deep import paths might get broken code after internal refactors rename files.
 
 As of [feat: add package.json exports for public packages](https://github.com/typescript-eslint/typescript-eslint/pull/6458), `@typescript-eslint/*` packages now use `exports` to prevent importing internal file paths.
-Developers must now mostly import directly from the package names, e.g.:
+Developers must now mostly import directly from the package names. For example, to fix the code from the previous example, you would use the following:
```

Comment:
> you

I think the `.e.g.` (or, if we don't like the abbreviation, a `for example`) says just about the same as this new sentence. Let's revert this change.

As an aside: for these general-purpose docs I don't think we want to phrase them as a personal journey. I in the past received feedback in -I think- the ESLint core repo that _"you"_ can come across as a bit pointed, even accusatory. I agree with that point of view.

Reply:
ok, i reverted it